### PR TITLE
feat(vpaas, recording): Show recording link to recording initiator

### PR DESCRIPTION
**Pull request by vp8x8**
This applies only for jaas users for now but is easily extensible.
Changed the recording sharing icon according to ui design.

### CODE REVIEW

File: react/features/base/config/functions.any.js [hunk 1]
```diff
@@ -40,6 +40,26 @@ export function createFakeConfig(baseURL: string) {
...
 }
 
+/**
+ * Selector used to get the meeting region.
+ *
+ * @param {Object} state - The global state.
+ * @returns {string}
+ */
+export function getMeetingRegion(state: Object) {
+    return state['features/base/config'].deploymentInfo.region;
```
Comment (damencho) [inline react/features/base/config/functions.any.js:50]: Can you make sure it will work without deploymentInfo in config.js

**Review by damencho — CHANGES_REQUESTED** [review]

File: react/features/base/config/functions.any.js (hunk 1 above)
Comment (vp8x8) [reply react/features/base/config/functions.any.js:50]: It's my understanding that we need the region in order to know where to look for the recording.

Is there any other place where we can get the meeting region from? Are there cases where this config info is missing?

Comment (damencho) [reply react/features/base/config/functions.any.js:50]: No, I mean this code will through an error if there is no region in the config. And the default config of people don't have that... 
Make it maybe to return empty string or something and to pass empty string to the request ... To make the code reusable and for other cases, but not totally breaking is what I meant ... Or at least safe guard it, and if there is no deploymentInfo skip everything with some error print ...

Comment (vp8x8) [reply react/features/base/config/functions.any.js]: Ok, got it.

File: react/features/recording/actions.any.js [hunk 2]
```diff
@@ -136,26 +142,68 @@ export function showStoppedRecordingNotification(streamType: string, participant
...
+                    dispatch(showErrorNotification({
+                        titleKey: 'recording.errorFetchingLink'
+                    }));
+
+                    return logger.error('Could not fetch recording link', err);
+                }
+            }
+        }
+
+        dispatch(showNotification(dialogProps));
```
Comment (hristoterezov) [inline react/features/recording/actions.any.js:205]: Did you remove the timeout on purpose?

**Review by hristoterezov — APPROVED** [review]

**Review by damencho — APPROVED** [review]

**Pull request merged**